spec/Records.tex

```diff
@@ -512,8 +512,8 @@ \subsection{The Method Receiver and the {\em this} Argument}
 \section{The {\em this} Method}
 \index{records!indexing}
 
-As with classes, records can be supplied with a \chpl{this} method.  This method
-defines the behavior of the indexing operator \chpl{[]}.
+As with classes, records can declare a \chpl{this} method.  Doing so
+permits record objects to be ``called''/''indexed.''
```

Comment:
Nick caught that earlier (see above), I commented on it (but forgot to push "comment"), fixed it (but then seemingly forgot to push it).  Argh.